repondeur/zam_repondeur/__init__.py

```diff
@@ -131,10 +134,26 @@ def get_user(request: Request) -> Optional[User]:
     user_id = request.unauthenticated_userid
     if user_id is not None:
         user: Optional[User] = DBSession.query(User).get(user_id)
+        if user is not None:
+            team = get_team(request)
+            if team is not None and team not in user.teams:
+                user.teams.append(team)
         return user
     return None
 
 
+def get_team(request: Request) -> Optional[Team]:
+    team_name = request.environ.get("REMOTE_USER")
+    if team_name is None:
+        return None
+    team: Optional[Team] = DBSession.query(Team).filter_by(name=team_name).first()
+    if team is None:
+        team = Team(name=team_name)
```

Comment:
Is that intentional not using the `.create()` pattern?

Reply:
Nah, we should be consistent and use create here too.